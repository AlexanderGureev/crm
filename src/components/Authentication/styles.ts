import styled from 'styled-components';
import { Form as AntdForm, Input as AntdInput } from 'antd';

export type StyledForm = typeof AntdForm & {
  Header: React.FC;
  Input: React.FC;
};

export const FormContainer = styled.div`
  background-color: var(--white);
  border-radius: 10px;
  padding: 100px;
  box-shadow: 0 20px 50px var(--gray1);
`;

export const FormHeader = styled.span`
  margin-bottom: 24px;
  text-align: center;
`;

export const Form: StyledForm = styled(AntdForm)`
  display: flex;
  justify-content: center;
  flex-direction: column;

  .ant-form-explain {
    padding: 5px 0;
    text-align: center;
  }
  .ant-form-item-with-help {
    margin-bottom: 10px;
  }
` as any;

export const Input = styled(AntdInput)`
  && {
    .ant-input {
      font-size: 16px;
    }
    .ant-input:not(:first-child) {
      padding-left: 35px;
    }
    height: 45px;
    border-radius: 4px;
  }
`;

Form.Header = FormHeader;
