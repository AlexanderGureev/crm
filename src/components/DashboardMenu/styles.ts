import styled from 'styled-components';
import { Menu as AntdMenu, Icon as AntdIcon } from 'antd';
import { MenuLink } from '../MenuLink';
import { MenuType } from './types';

export const Menu: MenuType = styled(AntdMenu)`` as any;
const MenuItem: MenuType['Item'] = styled(AntdMenu.Item)`` as any;

Menu.Item = MenuItem;
Menu.Link = styled(MenuLink)`
  && {
    position: absolute;
    top: 0;
    width: 100%;
    height: 100%;
    left: 0;
  }
`;
Menu.Text = styled.span`
  font-size: 16px;
  padding-left: 5px;
`;
Menu.Icon = styled(AntdIcon)`
  font-size: 20px !important;
`;
