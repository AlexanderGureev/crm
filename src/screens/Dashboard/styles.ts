import styled from 'styled-components';
import { Layout as AntdLayout, Icon } from 'antd';

export const DashboardContainer = styled.div``;
export const Layout: typeof AntdLayout = styled(AntdLayout)`
  && {
    height: 100vh;
  }
` as any;

export const Header = styled(AntdLayout.Header)`
  && {
    padding: 0 20px;
    background-color: var(--white);
  }
`;

export const CloseIcon = styled(Icon)`
  font-size: 24px;
`;

export const ContentContainer = styled(AntdLayout.Content)`
  margin: 24px 16px;
  padding: 24px;
  background: var(--white);
  min-height: 280px;
`;

export const Sider = styled(AntdLayout.Sider)``;
