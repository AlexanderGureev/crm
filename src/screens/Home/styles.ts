import styled from 'styled-components';

export const HomeContainer = styled.div`
  width: 100%;
  height: 100vh;
  background: linear-gradient(120deg, var(--bgGray1), var(--bgGray2));
  display: flex;
  justify-content: center;
  align-items: center;
`;
