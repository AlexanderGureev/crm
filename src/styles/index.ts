import styled from 'styled-components';
import { Button as AntdButton } from 'antd';

export const H2 = styled.h2`
  font-weight: 700;
  font-size: 26px;
  color: var(--gray7);
  margin: 0;
`;

export const Text = styled.span`
  color: var(--gray5);
`;

export const Button = styled(AntdButton)`
  && {
    padding: 5px 50px;
    height: 40px;
    font-size: 16px;
    border: none;
    box-shadow: 0 5px 25px var(--blue02);
  }
`;
