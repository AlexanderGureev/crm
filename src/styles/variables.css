:root {
  /* Fonts */
  --font-primary: 'Montserrat', 'MontserratLocal', sans-serif;
  --bgGray1: #dedfe4;
  --bgGray2: #e0e1e9;
  --white: #ffffff;
  --gray1: rgba(0, 0, 0, 0.1);
  --gray5: rgba(0, 0, 0, 0.5);
  --gray7: rgba(0, 0, 0, 0.7);
  --blue: #1233fc;
  --blue02: rgba(24, 144, 255, 0.2);
}
