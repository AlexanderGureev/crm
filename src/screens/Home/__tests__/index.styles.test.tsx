import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import { HomeContainer } from "../styles";

test("HomeContainer renders", () => {
  const tree = renderer.create(<HomeContainer />).toJSON();
  expect(tree).toMatchSnapshot();
});
