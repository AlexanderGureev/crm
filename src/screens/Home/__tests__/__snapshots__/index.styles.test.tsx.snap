// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeContainer renders 1`] = `
.c0 {
  width: 100%;
  height: 100vh;
  background: linear-gradient(120deg,var(--bgGray1),var(--bgGray2));
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
  className="c0"
/>
`;
